guides/source/action_controller_overview.md

```diff
@@ -158,13 +178,21 @@ So for example, if you are sending this JSON content:
 
 Your controller will receive `params[:company]` as `{ "name" => "acme", "address" => "123 Carrot Street" }`.
 
-Also, if you've turned on `config.wrap_parameters` in your initializer or called [`wrap_parameters`][] in your controller, you can safely omit the root element in the JSON parameter. In this case, the parameters will be cloned and wrapped with a key chosen based on your controller's name. So the above JSON request can be written as:
+#### Configuring Wrap Parameters
+
+You can [configure wrap parameters](configuring.html#config-action-controller-wrap-parameters-by-default) option if you want to omit the root element in the JSON parameters. It is `true` by default.
+
+```ruby
+config.action_controller.wrap_parameters_by_default = true
+```
+
+With this configuration, parameters will be cloned and wrapped with a key chosen based on your controller's name.
 
 ```json
 { "name": "acme", "address": "123 Carrot Street" }
 ```
 
-And, assuming that you're sending the data to `CompaniesController`, it would then be wrapped within the `:company` key like this:
+Assuming that you're sending the data to `CompaniesController`, the above JSON would be wrapped within the `:company` key like this:
```

Comment:
The example below is confusing for me because we see `name` and `address` both inside the company key but also as at that same level:

```
{ name: "acme", address: "123 Carrot Street", company: { name: "acme", address: "123 Carrot Street" } }
```

I don't think I fully understand this section on how to configure wrap parameters. Maybe it would also be good to  continue using the example from the previous section?